repose-aggregator/artifacts/docker/src/docker/centos/Dockerfile

```diff
@@ -29,5 +29,9 @@ RUN touch /var/log/repose/current.log
 # "-p" flag with the Docker run command.
 EXPOSE 8080
 
+# This environment variable is used to set command-line options.
+# The user can manually set these options using the "-e" flag with the Docker run command.
+ENV JAVA_OPTS=
```

Comment:
This instruction is not strictly required. I added it to make it clear that we can handle a `JAVA_OPTS` environment variable, and in the future, we can provide a default value... but for now, it does nothing.